### DISCUSSION TITLE
Extension Marketplace

### DISCUSSION THREAD

**Discussion by mcamendoza1** [directus/directus#14063] [Archive]
I will make directus more flexible if there will be a marketplace for extension and verified by the community
https://github.com/directus/directus/discussions?discussions_q=Extension%20Marketplace

**Comment by benhaynes** [comment]
Absolutely — this is our focus for the second half of this year. Really exciting!

**Comment by rijkvanzanten** [comment]
Duplicate of https://github.com/directus/directus/discussions/2820
